src/mlpack/core/data/load_csv.hpp

```diff
@@ -94,7 +166,21 @@ class LoadCSV
     std::string line;
     while (std::getline(inFile, line))
     {
-      ++rows;
+       ++rows;
```

Comment:
```suggestion
      ++rows;
```
I think this has one too many spaces.  (It should be 2-space tabs.) :+1: